.github/workflows/coverage.yml

```diff
@@ -45,7 +45,6 @@ jobs:
             -t lcov \
             --branch \
             --ignore-not-existing \
-            --ignore 'crates/topos-api/src/grpc/generated/*' \
```

Comment:
Ah good catch but we may need to keep this but change the line to `crates/topos-core/src/api/grpc/generated/*`